os/pm/pm_metrics.c

```diff
@@ -290,6 +360,9 @@ int pm_metrics(int milliseconds)
 		pm_metrics_update_domain(index);
 		g_pm_metrics->domain_metrics.stime[index] = start_time;
 	}
+	for (index = 0; index < CONFIG_MAX_TASKS; index++) {
+		g_pm_metrics->thread_metrics.min_sleep_ticks[index] = UINT32_MAX;
+	}
```

Comment:
Do we need to initialize wakeup_counts also?

Reply:
No, `pm_alloc`  already initializing it with `0`.